decidim-core/app/models/decidim/messaging.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Messaging
+    def self.table_name_prefix
+      "decidim_messaging_"
```

Comment:
Is this needed?

Reply:
No. It simplifies associations inside the namespace so one does not need to specify class names and foreign keys if they follow conventions. But it's not used anywhere else, so I should probably remove it, right?

Reply:
Hei, so I had at look into this and it is actually necessary, so that the table names of the different models are properly resolved.

At this point we could do several things:

* Don't namespace table names. That means changing the migrations so that the tables are named with just the prefix `decidim_` instead of `decidim_messaging_`.

* Move everything to a separate engine. In that case, doing `isolate_namespace Decidim::Messaging` in the engine definition would accomplish the same thing I'm doing here.

* Leave it as it is.

I vote for the last one :)